### PR TITLE
suricata 3.2.2

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Race condition seems to have been fixed. https://redmine.openinfosecfoundation.org/issues/1470

`audit` complained about `pypi.python.org`, changed to `files.pythonhosted.org`

### CODE REVIEW

File: Formula/suricata.rb [hunk 1]
```diff
@@ -25,19 +25,16 @@ class Suricata < Formula
   depends_on "hiredis" => :optional
 
   resource "argparse" do
-    url "https://pypi.python.org/packages/source/a/argparse/argparse-1.3.0.tar.gz"
+    url "https://files.pythonhosted.org/packages/source/a/argparse/argparse-1.3.0.tar.gz"
```
Comment (fxcoudert) [inline Formula/suricata.rb]: Can this be updated to argparse 1.4.0?

File: Formula/suricata.rb [hunk 2]
```diff
@@ -25,19 +25,16 @@ class Suricata < Formula
...
     sha256 "b3a79a23d37b5a02faa550b92cbbbebeb4aa1d77e649c3eb39c19abf5262da04"
   end
 
   resource "simplejson" do
-    url "https://pypi.python.org/packages/source/s/simplejson/simplejson-3.6.5.tar.gz"
```
Comment (fxcoudert) [inline Formula/suricata.rb]: Update to simplejson 3.11.1 if possible

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
Thanks @commitay! Please squash this down to 1 commit as well.

**Review by fxcoudert — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for the PR @commitay! Shipped!

**Comment by commitay** [comment]
Thanks!

**Pull request closed without merging**